app/javascript/components/student/tracks-list/StatusFilter.jsx

```diff
@@ -7,17 +7,21 @@ export function StatusFilter({ options, dispatch, ...props }) {
   }
 
   return (
-    <div>
+    <div className="status-filters">
       {options.map((option) => {
+        // TODO: Populate .count once these
+        // filters are moved client side
         return (
           <button
             key={option.value}
+            className={option.value === value ? 'selected' : ''}
             disabled={option.value === value}
             onClick={(e) => {
               handleChange(e, option.value)
             }}
           >
             {option.label}
+            <span className="count">20</span>
```

Comment:
See comment above - this will need to be implemented once these track filters are moved inline.